pyrtlnet/numpy_inference.py

```diff
@@ -230,32 +229,36 @@ def preprocess_image(self, test_image: np.ndarray) -> np.ndarray:
         #
         # Adding input_zero_point (-128) effectively converts the uint8 image data to
         # int8, by shifting the range [0, 255] to [-128, 127].
-        return np.reshape(
-            test_image / self.input_scale + self.input_zero, newshape=flat_shape
-        ).astype(np.int8)
+
+        #Flatten 2D -> 1D
+        return test_image.reshape(test_image.shape[0], -1).T
```

Comment:
The rest of the code in this file prefers `.transpose()` over `.T`, so we should be consistent here